pks-pipelines/manifest.yml

```diff
@@ -0,0 +1,65 @@
+name: ((deployment-name))
+
+releases:
+  - name: kubo
+    version: ((kubo-version))
+  - name: docker
+    version: ((docker-version))
+#  - name: cfcr-etcd
+#    version: ((etcd-version))
```

Comment:
Not sure whether these lines should be (a) removed, (b) uncommented, or (c) left in with comments..?